spectre-meltdown-checker.sh

```diff
@@ -273,12 +335,12 @@ fi
 
 if [ "$opt_live" = 1 ]; then
 	if [ "$(id -u)" -ne 0 ]; then
-		_echo "\033[31mNote that you should launch this script with root privileges to get accurate information.\033[0m"
-		_echo "\033[31mWe'll proceed but you might see permission denied errors.\033[0m"
-		_echo "\033[31mTo run it as root, you can try the following command: sudo $0\033[0m"
-		_echo
+		_warn "Note that you should launch this script with root privileges to get accurate information."
+		_warn "We'll proceed but you might see permission denied errors."
+		_warn "To run it as root, you can try the following command: sudo $0"
+		_warn
 	fi
-	_echo "Checking for vulnerabilities against live running kernel \033[35m"$(uname -s) $(uname -r) $(uname -v) $(uname -m)"\033[0m"
+	_verbose "Checking for vulnerabilities against live running kernel \033[35m"$(uname -s) $(uname -r) $(uname -v) $(uname -m)"\033[0m"
```

Comment:
I agree with the fact that the messages `Will use vmlinux/sysmap/kconfig ...` can require a `-v`

However I think we should keep the `uname -a` in the default output: when people copy/paste the output from the script, it'll then be of some use to others. Without the `uname -a`, the context is lost and the script output becomes useless except if you're the one who launched it.

For a shorter output, we could split `Checking for vulnerabilities against live running kernel` (and show it only under -v). And keep the actual `uname -a` by default